modules/openthread/CMakeLists.txt

```diff
@@ -234,6 +234,52 @@ list(APPEND ot_libs openthread-mtd)
 endif()
 endif()
 
+if(CONFIG_OPENTHREAD_HDLC_RCP)
+set(ot_host_inc
+    ${ZEPHYR_BASE}/include
+    ${ZEPHYR_BASE}/kernel/include
+    ${ZEPHYR_BASE}/arch/arm/include
+    ${ZEPHYR_BASE}/../modules/lib/openthread/src
+    ${ZEPHYR_BASE}/../modules/lib/openthread/src/core
+    ${ZEPHYR_BASE}/../modules/lib/openthread/third_party/mbedtls/repo/include
```

Comment:
Zephyr is using it's own mbed TLS: https://github.com/zephyrproject-rtos/mbedtls not exactly same as the on in OT.

Reply:
resolved